openlibrary/templates/lib/nav_head.html

```diff
@@ -77,8 +81,11 @@
       </div>
     </li>
     <li class="more-menu">
-      <a>$_('More') <img class="down-arrow"
-        width="7" height="4" aria-hidden="true" alt="" role="presentation" src="/static/images/down-arrow.png"/></a>
+      <label role="button" for="toggle-more-menu" class="more-menu-button">$_('More') 
```

Comment:
Suggested follow up: Given the repetition could we use a macro in the templates for each of these?